index.js

```diff
@@ -102,9 +102,9 @@ module.exports = function (content) {
     dest: '',
     writeFiles: false,
     formatOptions: config.formatOptions || {
-        ttf: {
-            ts: 0 // it's important to set timestamp to fixed value for ttf generation to have consistent output in long term with the same hash
-        }
+      ttf: {
```

Comment:
Its possible to set it in config, but the default settings should be correct.
The font file binaries generated by this loader must not depend on generation time. Without this option, the TTF file generated by svg2ttf package contains current date/time by default, so hash of two TTF files generated in different time from same source differs. It makes problems when using [hash] as a part of font filename.